openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -3198,7 +3233,8 @@ Error AMDGPUKernelTy::launchImpl(GenericDeviceTy &GenericDevice,
 
   // Push the kernel launch into the stream.
   return Stream->pushKernelLaunch(*this, AllArgs, NumThreads, NumBlocks,
-                                  GroupSize, StackSize, ArgsMemoryManager);
+                                  GroupSize, static_cast<uint32_t>(StackSize),
```

Comment:
Change it to uint32_t everywhere instead. Let's assume we stop at 4GB stack size.

Reply:
Thanks -- Sure, sounds reasonable.